src/ast/semantic_analyser.cpp

```diff
@@ -2121,23 +2124,27 @@ void SemanticAnalyser::visit(AttachPoint &ap)
       }
     }
   }
-  else if (ap.provider == "kfunc" || ap.provider == "kretfunc")
+  else if (ap.provider == "kfunc" || ap.provider == "kretfunc" ||
+           ap.provider == "lsm")
   {
 #ifndef HAVE_BCC_KFUNC
     error("kfunc/kretfunc not available for your linked against bcc version.",
```

Comment:
I think "lsm not available ..." is better when lsm is not available since there is a case where kfunc is supported but lsm is not.

Reply:
yes, will change